app/Http/Controllers/HomeController.php

```diff
@@ -0,0 +1,26 @@
+<?php
+
+namespace App\Http\Controllers;
+
+class HomeController extends Controller
+{
+    /**
+     * Create a new controller instance.
+     *
+     */
+    public function __construct()
+    {
+        $this->middleware('auth');
```

Comment:
This was fixed by this commit [d4665da](https://github.com/cham11ng/multi-tenant-postgres/pull/1/commits/d09b09d8a2e9dc5205b91946e64586aabe5ebe83)